example/package.json

```diff
@@ -14,9 +14,9 @@
     "start": "electron ."
   },
   "dependencies": {
-    "about-window": "*"
+    "about-window": "1.4.0"
```

Comment:
No, there's no reason, because it's better to put the local module (path) instead of version number (if we want to test our modifications). From npm 2.0.0 we can do that. http://stackoverflow.com/questions/14381898/local-dependency-in-package-json